guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,12 +4,65 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== New All Hosts index page improvements
+
+There are some significant improvements in the experimental new All Hosts index page with this release.
+To use the new page by default, in Settings, set Show new host overview page to Yes.
+
+*Host bulk actions* - You can now select one or more hosts and click the vertical ellipsis menu to perform the following actions:
+
+* Manage content
+* Packages (install / remove / upgrade)
+* Errata
+* Content source
+* Content view environments - change the content view and lifecycle environment of hosts
+
+*Single-host actions* - In addition to actions added by Foreman, you can click the vertical ellipsis in the rightmost table column (without selecting the host) to change content view environments on a single host.
+
+*Column selector* - You can now click Manage Columns and customize columns displayed in the table.
+The column options are the same as those on the legacy All Hosts page. 
+Any changes made in either the new experimental *All Hosts* page and the legacy *All Hosts* page reflect on each other.
+
+=== Assigning multiple content view environments to hosts
```

Comment:
I think this reads better, but I'm looking at @bangelic for a definitive answer
```suggestion
=== Assign multiple content view environments to hosts
```

Reply:
I agree with @ekohl